integration-tests/tests/src/tests/enums.ts

```diff
@@ -84,4 +85,13 @@ describe("Enums", function () {
       });
     });
   });
+  describe("WaitForSync", function () {
```

Comment:
Is this part of CJS?

Reply:
This is me forgetting to add this enum test when doing the Flx Sync v2 PR 😉